### PR TITLE
implemented EngineToolDTO

### PR DESCRIPTION
**Pull request by plata**

### CODE REVIEW

File: phoenicis-engines/src/main/java/org/phoenicis/engines/dto/EngineToolDTO.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.apache.commons.lang.builder.ToStringBuilder;
+import org.phoenicis.configuration.localisation.Translatable;
+import org.phoenicis.configuration.localisation.TranslatableBuilder;
+import org.phoenicis.configuration.localisation.Translate;
+
+@JsonDeserialize(builder = EngineToolDTO.Builder.class)
+@Translatable
+public class EngineToolDTO {
```
Comment (madoar) [inline phoenicis-engines/src/main/java/org/phoenicis/engines/dto/EngineToolDTO.java]: JavaDoc

File: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/WinePrefixContainerWineToolsTab.java [hunk 2]
```diff
@@ -54,29 +86,17 @@ private void populate() {
...
-        });
-
-        Button rebootWindows = new Button(tr("Windows reboot"));
-        rebootWindows.getStyleClass().addAll("toolButton", "rebootWindows");
-        rebootWindows.setOnMouseClicked(event -> {
-            this.lockAll();
-            winePrefixContainerController.runInPrefix(container, "RebootWine", this::unlockAll,
-                    e -> Platform.runLater(() -> new ErrorMessage("Error", e).show()));
-        });
+        for (EngineToolDTO tool : tools) {
```
Comment (madoar) [inline phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/WinePrefixContainerWineToolsTab.java]: Why not use `MappedList` and `Bindings#bindContent`?

Comment (plata) [reply phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/WinePrefixContainerWineToolsTab.java]: Not sure how you would like to have it. Can we merge it and you can do the change afterwards?

Comment (madoar) [reply phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/WinePrefixContainerWineToolsTab.java]: Basically you could use:

```java
MappedList<...> toolButtons = new MappedList<>(FXCollections.observableArrayList(tools), tool -> {
   ...
   return toolButton; 
});
```
Afterwards you can use 
```java
Bindings.bindContent(toolsContentPane.getChildren(), toolButtons);
```

Comment (plata) [reply phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/WinePrefixContainerWineToolsTab.java]: Ok. What's the advantage?

Comment (madoar) [reply phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/WinePrefixContainerWineToolsTab.java]: It's shorter and it fits better to the way our tabs are created, right?

Comment (plata) [reply phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/WinePrefixContainerWineToolsTab.java]: Shorter I don't know. But the second point is true.

**Pull request merged**